docs/input-formats.md

```diff
@@ -25,11 +25,14 @@ To access data remotely, create a file ending in `.`_format_`-remote` as follows
     {
        "authentication": null,
        "url": "http://some.url/format/endpoint",
-       "ttl": 10
+       "ttl": 10,
+       "timeout": 60
```

Comment:
Just looking at the json, I'd assume this is seconds, especially with the example of 60 (is setting this to an hour reasonable?). Might want to clarify the field name and/or the example.